src/test/crimson/seastore/test_omap_manager.cc

```diff
@@ -137,17 +137,56 @@ struct omap_manager_test_t :
     test_omap_mappings.erase(test_omap_mappings.find(key));
   }
 
+  std::vector<std::string> rm_key_range(
+    omap_root_t &omap_root,
+    Transaction &t,
+    const std::string &first,
+    const std::string &last) {
+    logger().debug("rm keys in range {} ~ {}", first, last);
+    auto config = OMapManager::omap_list_config_t();
```

Comment:
Follow the convention to use meaningful factory method from `omap_list_config_t` for readability.